### PR TITLE
Express notes API | Jaime Sandoval

### PR DESCRIPTION
**Pull request by JaimeGSandoval**

### CODE REVIEW

File: express-notes-api/index.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const data = require('./data.json');
+let stringifiedNoteData = null;
```
Comment (thebearingedge) [inline express-notes-api/index.js:6]: You should declare this variable down where you need it.

File: express-notes-api/index.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+app.get('/api/notes', function (req, res) {
+  const notes = [];
+  const notesKeyLength = Object.keys(data.notes).length;
+  if (notesKeyLength === 0) {
+    res.send(notes);
+  } else {
+    for (const i in data.notes) {
+      notes.push(data.notes[i]);
+    }
+    res.send(notes).json();
```
Comment (thebearingedge) [inline express-notes-api/index.js:27]: We normally do this as `res.json(notes)`. `.send()` works, but it's better to use the more specific method. Please replace your `.send()` calls with `.json()`

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, good job so far. Just a little cleanup needed.

File: express-notes-api/index.js (hunk 2 above)
Comment (JaimeGSandoval) [reply express-notes-api/index.js:27]: Okay, will do. I get mixed up with which one to use. Thanks.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**